docker/templates/README.md

```diff
@@ -0,0 +1,79 @@
+# 🍪 Containerization templates
+
+This directory contains [cookiecutters](https://github.com/audreyr/cookiecutter)
+for the containerization of web applications.
+
+## Getting started
+
+### 1. Set up.
+
+Install `cookiecutter` in your virtual environment of choice. (It's a [great
+candidate for `gus`](https://github.com/datamade/ops/wiki/How-to-DataMade-(Resources-for-new-hires)#python)!)
```

Comment:
What if we... containerized the containerization templating process 🤔

Reply:
maximum recursion depth reached

Reply:
jk i think this is an interesting idea! let me look into it!